### DISCUSSION TITLE
TypeError: Cannot read property 'prototype' of undefined

### DISCUSSION THREAD

**Discussion by Vikash-8090-Yadav** [NomicFoundation/hardhat#3551] [Troubleshooting]
Greetings!
When i am running  this command  `npx hardhat run scripts/deploy.js`  am getting this error `TypeError: Cannot read property 'prototype' of undefined` . If someone knows please help ! 🙏

deploy.js

const hre = require("hardhat");
  async function main() {
  const VikashPR = await hre.ethers.getContractFactory("VikashPR");
  const vikashpr = await VikashPR.deploy();
  await vikashpr.deployed();
}
// We recommend this pattern to be able to use async/await everywhere
// and properly handle errors.
main()
  .then(()=>process.exit(0))
  .catch((error)=>{
    console.error(error);
    process.exit(1);
  });

Error : 

![Screenshot from 2023-01-08 16-38-39](https://user-images.githubusercontent.com/85225156/211192796-49754a0a-4487-4afb-952e-f4ce72e933c4.png)

**Comment by alcuadrado** [comment]
Can you please open an issue with the steps to reproduce it instead of a discussions? Thanks!
